### PR TITLE
Added a Code of Conduct

### PR DESCRIPTION
**Pull request by TheBusyBiscuit**
## Description

This Pull Request adds a Code of Conduct to this repository.

I stuck to the widely used "Contributor Covenant Code of Conduct" but made large modifications to it.
I think this helps us communicate the way we want the community to behave and moderated and it is definitely a good thing to have.

@WalshyDev please review.

### CODE REVIEW

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+### What is a Code of Conduct?
+> A code of conduct is a document that establishes expectations for behavior for your project’s participants. 
+> Adopting, and enforcing, a code of conduct can help create a positive social atmosphere for your community.
+<br>\- *Source: [https://opensource.guide](https://opensource.guide/code-of-conduct/#why-do-i-need-a-code-of-conduct)*
+
+This document should serve the purpose of outlining the behaviour we expect from any participant of the project.<br>
+
+## Scope
+This Code of Conduct applies to all sections of the [Slimefun4 GitHub repository](https://github.com/TheBusyBiscuit/Slimefun4), our [Slimefun GitHub organization](https://github.com/Slimefun) and all repositories owned by said organization.<br>
+For our official discord server, please refer to our article on [Discord Rules](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Discord-Rules).
```
Comment (WalshyDev) [inline .github/CODE_OF_CONDUCT.md:11]: Capitalise Discord

File: .github/CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+Everyone who engages with this project on any of these repositories is expected to follow the Code of Conduct.<br>
+This includes maintainers, contributors, sponsors and anyone who engages in the "Issues" section on GitHub.
+
+## Our Standards
+Examples of behavior that contributes to a positive environment for our
+community include:
```
Comment (WalshyDev) [inline .github/CODE_OF_CONDUCT.md:18]: why was this put on another line? Doesn't matter as GH will have it on the same one but still

File: .github/CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+* Demonstrating empathy and kindness toward other people
```
Comment (WalshyDev) [inline .github/CODE_OF_CONDUCT.md:20]: *towards

**Review by WalshyDev — CHANGES_REQUESTED** [review]
Looking good

File: .github/CODE_OF_CONDUCT.md (hunk 3 above)
Comment (TheBusyBiscuit) [reply .github/CODE_OF_CONDUCT.md:20]: I knew it. But it's the standard text so I didn't dare to change it.

File: .github/CODE_OF_CONDUCT.md (hunk 2 above)
Comment (TheBusyBiscuit) [reply .github/CODE_OF_CONDUCT.md:18]: Ask the Contributor Covenant, not me.

**Comment by TheBusyBiscuit** [comment]
Made changes and added a new section too

**Comment by WalshyDev** [comment]
LGTM so far.

**Pull request merged**